src/libaktualizr/package_manager/ostreemanager_test.cc

```diff
@@ -212,7 +212,15 @@ int main(int argc, char **argv) {
     std::cerr << "Error: " << argv[0] << " requires the path to an OSTree sysroot as an input argument.\n";
     return EXIT_FAILURE;
   }
-  test_sysroot = argv[1];
+
+  TemporaryDirectory temp_sysroot;
+  test_sysroot = temp_sysroot / "sysroot";
+  // uses cp, as boost doesn't like to copy bad symlinks
```

Comment:
What's wrong with the symlink? Why is it bad?

Reply:
To be specific, in `ostree_repo/ostree/deploy/dummy-os/var`, there is a `run` symlinking to `../run` which doesn't exist. That's the internal to ostree so it seems to be working fine but when using `Utils::copyDir`, the boost function that we use fails.

I believe Serhiy had a similar issue in the past and used the same solution.

Reply:
Hmm. Ugly. Oh well, do what you gotta do.